models/Product-types.js

```diff
@@ -0,0 +1,57 @@
+'use strict'; 
+const sqlite3 = require('sqlite3').verbose();
+const db = new sqlite3.Database('bangazon.sqlite');
+
+module.exports.getAllProdTypes = () => {
+  console.log('getTypes called');
```

Comment:
Hello Hunter. Hope your Friday is going well. I know you are on lunch, but I would like to request that you remove this console.log as soon as possible so we can move forward with merging this PR. See you back at the office

- Eli